students/16th/donghalee/styles/login.css

```
@@ -0,0 +1,223 @@
+    align-items: center;
+}
+
+.box {
+    width: 350px;
+    border: solid 1px rgba(var(--b6a,219,219,219),1);
+    border-radius: 1px;
+    background-color: white;
+}
+
+h1 {
+    font-family: 'Lobster', cursive;
+    font-size:  30px;
+    text-align: center;
+    height: 51px;
+    width: 175px;
+    margin: 22px auto 12px;
+}
+
+form {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+input  {
+    width: 268px;
+    height: 38px;
+    border: 1px solid rgba(var(--ca6,219,219,219),1);
+    border-radius: 3px;
+    background-color: rgba(var(--b3f,250,250,250),1);
+    margin: 0 40px 6px;
+    padding: 9px 0 7px 8px;
+}
+
+.loginBtn {
+    width: 268px;
+    height: 38px;
+    border: 1px solid #B9DFFC;
+    border-radius: 3px;
+    margin: 8px 40px;
+    background-color: #B9DFFC;
+    opacity: 0.7;
+    color: white;
+}
+
+.and {
+    width: 350px;
+    height: 30px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 10px 40px 10px;
+}
+
+.meanlessLine {
+    height: 1px;
+    background-color: rgba(var(--b38,219,219,219),1);
+    width: 104px;
+    margin: auto 0px;
+}
+
+
+.pointLine {
+    width: 30px;
+    color: rgba(var(--f52,142,142,142),1);
+    font-size: 13px;
+    line-height: 30px;
+    font-weight: 600;
+    flex-shrink: 0;
+    text-align: center;
+}
+
+.faceBtn {
+    width: 268px;
+    height: 20px;
+    cursor: pointer;
+    text-align: center;
+    border: 0;
+    outline: 0;
+    color: #385185;
+    line-height: 16px;
+    font-size: 16px;
+    font-weight: 600;
+    background-color:transparent
+}
+
+img {
+    margin: auto;
+}
+
+.fbLog {
+    text-align: center;
+}
+
+.forgotPassword {
+    width: 350px;
+    height: 14px;
+    font-size: 12px;
+    line-height: 14px;
+    text-align: center;
+    color: #00376b;;
+    text-decoration: none;
+    margin: 24px 0;
+    cursor: pointer;
+}
+
+.join {
+    width: 350px;
+    height: 63px;
+    border: solid 1px rgba(var(--b6a,219,219,219),1);
+    border-radius: 1px;
+    background-color: white;
+    font-size: 15px;
+    text-align: center;
+    line-height: 63px;
+    margin: 10px 0;
+}
+
+.makeId {
+    text-decoration: none;
+    color: #0095f6;
+    font-weight: 600;
+}
+
+.app {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+footer {
+    width: 100%;
+    height: 120px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    
+}
+
+.banner {
+    width: 100;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #8e8e8e;
+}
+
+.lan {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.firstLine {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
+
+.secondLine {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+}
```

```suggestion
.firstLine,
.secondLine {
    display: flex;
    flex-direction: row;
    align-items: center;
    justify-content: center;
}
```
같은 속성을 공유하고 있을 경우 묶어서 지정이 가능합니다 :)

---

반영했습니다 !